packages/api/cms-api/src/file-uploads/file-uploads-download.controller.ts

```diff
@@ -74,6 +74,7 @@ export function createFileUploadsDownloadController(options: { public: boolean }
                 "content-type": file.mimetype,
                 "last-modified": file.updatedAt?.toUTCString(),
                 "content-length": file.size,
+                "cache-control": options.public ? "max-age=86400, public" : "max-age=31536000, private", // same as DAM
```

Comment:
The generated download URLs have a 1 h timeout. Shouldn't this be reflected here?

Reply:
Options:
1. Calculate the remaining time and set it as max-age for both
2. Set no-store (like we did for download folders)
3. Set private and allow infinite caching

I'm not sure which option is the best, but all are fine probably.

Reply:
If this should be cached by CDN: :one:

if we don't care about cdn caching: :two:

As createDownloadUrl create a new, unique url for every usage, CDN cache can't be used anyway, It would just fill up with "garbage". For that reason :two: (or :three: is also ok)


(and the same should be applied to image function below)